test/cypress/integration/group-proposals.spec.js

```diff
@@ -236,33 +236,54 @@ describe('Proposals - Add members', () => {
       })
   })
 
-  it('It should display that no one is doing any contribution', () => {
+  it('User click on contribution page and see the card', () => {
     cy.getByDT('contributionsLink').click()
     cy.getByDT('addIncomeDetailsFirstCard').should('contain', 'Add your income details')
+  })
+
+  it('Open the income detail modal should ask only if you want to receive or pledge', () => {
```

Comment:
tests language/person should be consistent. in our case, avoid using "you" and stick to the 3rd person.
> Opening income detail modal for the 1st time should only ask the user's income.